docs/overview_of_strategies.rst

```diff
@@ -6,32 +6,49 @@ Axelrod's first tournament
++--------------------------+-------------------------------------------+--------------------------+
+| `Tideman and Chieruzzi`_ | Tideman and Chieruzzi (authors' names)    | Not Implemented          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Nydegger`_              | Nydegger (author's name)                  | Not Implemented          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Grofman`_               | Grofman (author's name)                   | :code:`Grofman`          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Shubik`_                | Shubik (author's name)                    | :code:`Shubik`           |
++--------------------------+-------------------------------------------+--------------------------+
+| `Stein and Rapoport`_    | Stein and Rapoport (authors' names)       | Not Implemented          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Grudger`_               | Grudger (by Friedman)                     | :code:`Grudger`          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Davis`_                 | Davis (author's name)                     | :code:`Davis`            |
++--------------------------+-------------------------------------------+--------------------------+
+| `Graaskamp`_             | Graaskamp (author's name)                 | Not Implemented          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Downing`_               | Downing (author's name)                   | Not Implemented          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Feld`_                  | Feld (author's name)                      | :code:`Feld`             |
++--------------------------+-------------------------------------------+--------------------------+
+| `Joss`_                  | Joss (author's name)                      | :code:`Joss`             |
++--------------------------+-------------------------------------------+--------------------------+
+| `Tullock`_               | Tullock (author's name)                   | :code:`Tullock`          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Unnamed Strategy`_      | Unnamed Strategy (by a Grad Student in    | Not Implemented          |
+|                          | Political Science)                        |                          |
++--------------------------+-------------------------------------------+--------------------------+
+| `Random`_                | Random                                    | :code:`Random`           |
```

Comment:
I think the error @lvisintini meant was that upon building the Random strategy does not link correctly. I thought it was not in the file but checked and I was wrong. I'm not sure what is happening here.